src/vm/gdbjit.cpp

```diff
@@ -1122,7 +1128,18 @@ void TypeMember::DumpDebugInfo(char* ptr, int& offset)
 void TypeMember::DumpStaticDebugInfo(char* ptr, int& offset)
 {
     const int ptrSize = sizeof(TADDR);
-    int bufSize = 0;
+    bool isValueType = m_member_type->GetTypeHandle().GetSignatureCorElementType() ==
+                            ELEMENT_TYPE_VALUETYPE;
+    int bufSize;
+    if (isValueType)
+    {
+        bufSize = ptrSize + 6;
```

Comment:
When the bufSize was computed right before the buffer got filled, it was ok to use just a number as the constant. Now that the places of the two usages of ptrSize + 6 and ptrSize + 2 are apart, could you please introduce named constants for each of these? Something along the lines of `ValueTypeBufSize` and `RefTypeBufSize`.

Reply:
Done.